debug/org.eclipse.cdt.debug.core/src/org/eclipse/cdt/debug/core/model/ITargetProperties.java

```diff
@@ -37,4 +37,4 @@ public interface ITargetProperties {
 	 * @param listener a property change listener
 	 */
 	void removePropertyChangeListener(Preferences.IPropertyChangeListener listener);
-}
+}
```

Comment:
Please undo this change, because it requires bundle bump in CDT for nothing.

Reply:
Done.